mesonbuild/environment.py

```diff
@@ -603,6 +603,8 @@ def __init__(self, source_dir: T.Optional[str], build_dir: T.Optional[str], opti
         exe_wrapper = self.lookup_binary_entry(MachineChoice.HOST, 'exe_wrapper')
         if exe_wrapper is not None:
             self.exe_wrapper = ExternalProgram.from_bin_list(self, MachineChoice.HOST, 'exe_wrapper')
+            if not self.exe_wrapper.found():
+                raise MesonException(f'Executable wrapper {self.exe_wrapper.name} not found.')
```

Comment:
Note that we have various places where we check if exe_wrapper is found, e.g. https://github.com/mesonbuild/meson/blob/master/mesonbuild/compilers/mixins/clike.py#L151. Is it intentional to try to configure with a not-found exe-wrapper? IMHO if it's set in the cross file we should not silently ignore it...

Reply:
Dropped that because it's unrelated and we should merge this PR in 1.2.1 ideally.